e2e-tests/cypress/support/step_definitions/send_coin_steps.ts

```diff
@@ -0,0 +1,69 @@
+import { And, Then, When } from '@badeball/cypress-cucumber-preprocessor'
+import { SendPage } from '../../e2e/models/SendPage'
+
+const sendPage = new SendPage()
+
+When(
+  'the user fills the send form with {string} {string} {string}',
+  (email: string, amount: string, memoText: string) => {
+    sendPage.enterReceiverEmail(email)
+    sendPage.enterAmount(`${amount}`)
```

Comment:
For coherence with the other:
```suggestion
    sendPage.enterAmount(amount)
```